NEWS

```diff
@@ -57,6 +57,13 @@ NEWS - user visible changes				-*- outline -*-
 ** New option -fdefault-file-colseq to specify the default
    file collating sequence
 
+** New dependency options: output COPY dependencies of COBOL files with
+   -M to only print dependencies without compiling, -MD to print
+   dependencies which compiling, -MP to add phony -targets, MQ to
+   quote Makefile-specific characters and -MG to keep -missing
+   copybooks. More specific is -fcopybook-deps to print only dependency
```

Comment:
Are there any "prints"? I've thought this is output commonly going to a dependency file`, with possibly the special case of output to std...out?
I think it is best to only note the options here or even say "related to -MT"( or similar) and then reference the new chapter in the manual to be more concise.

Reply:
I guess it's "print" in a wider definition;, whether to a file or to stdout (depends if using the `-MF` option).

Reply:
As noted, please change it to only have an "overview" here and point to the manual for the details.